requirements/optional-private.txt

```diff
@@ -3,7 +3,7 @@
 # git config --global url.https://<private hostname>/.insteadOf https://private.repo/
 
 git+https://private.repo/CFPB/agreement_database.git@2.2.7#egg=agreement-database
-git+https://private.repo/CFPB/cfgov-selfregistration.git@v1.2#egg=selfregistration
+selfregistration==@1.3.1
```

Comment:
Same as above-- we'll probably need to bake these instructions more fully into our docs:
[GHE]/CFGOV/platform/wiki/Using-our-Python-package-repository